include/fastdds/dds/topic/ContentFilteredTopic.hpp

```diff
@@ -85,10 +130,6 @@ class ContentFilteredTopic : public TopicDescription
      */
     virtual DomainParticipant* get_participant() const override;
 
-    /**
-     * @brief Getter for the TopicDescriptionImpl
-     * @return pointer to TopicDescriptionImpl
-     */
     TopicDescriptionImpl* get_impl() const override;
```

Comment:
Maybe we should consider making the base method and the override implementations protected.

Reply:
Having the implementation getter public makes no harm, as the user does not have the implementation class declaration.

Making this a protected method will break API, and require adding several friend classes.